tests/test_all_fields.py

```diff
@@ -55,10 +55,11 @@
 
 FIELDS_TO_NOT_CHECK = {
     'customers': {
-        # As per stripe documentation(https://stripe.com/docs/api/customers/object) `subscriptions` and `sources` fields
-        # are not included by default.
+        # As per stripe documentation(https://stripe.com/docs/api/customers/object) `subscriptions`, `sources` and `tax_ids` fields
+        # are not included by default in API response.
         'subscriptions',
```

Comment:
@prijendev Please update the code to execute the API call which will return maximum fields from the catalog.
This will help us reduce the warning messages during test cases execution

Reply:
Updated